### ISSUE TITLE
translator notes on words

### ISSUE THREAD

**Issue by arrocke** [arrocke/gloss-translation#310]
* sidebar has notes tab
* notes tab has rich text editor for translator notes if the current user is a language translator, otherwise it is hidden
* note is saved to database with edited timestamp and author
* note shows last timestamp and author

**Comment by Pertempto** [comment]
Blocked by rich text editor issue?

**Comment by Pertempto** [comment]
Or maybe we don't need a separate issue for that, it could just be implemented in two MRs for this issue. The first MR for the notes tab containing rich text editor, second MR for hooking up everything else.

**Comment by Pertempto** [comment]
Any implementation ideas for the tab bar? We probably want to roll our own, instead of using a library?

**Comment by arrocke** [comment]
Actually, we need to do #303 first. I shouldn't have moved this into ready. I'll throw some notes about the tab bar in that issue
